arm_joint_names:
  - elbow_joint
  - shoulder_lift_joint
  - shoulder_pan_joint
  - wrist_1_joint
  - wrist_2_joint
  - wrist_3_joint

arm_initial_positions:
  - 1.2
  - -0.75
  - 0
  - 1.127
  - 1.58
  - 1.51

gripper_joint_names:
  - finger_joint

gripper_open_positions:
  - 0

gripper_close_positions:
  - 0.1